docs/src/complex.md

```diff
@@ -2,7 +2,7 @@
 
 Complex numbers add some difficulty to the idea of a "gradient". To talk about `gradient(f, x)` here we need to talk a bit more about `f`.
 
-If `f` returns a real number, things are fairly straightforward. For ``c = x + yi`` and  ``z = f(c)``, we can define the adjoint ``\bar c = \frac{\partial z}{\partial x} + \frac{\partial z}{\partial y}i = \bar x + \bar y i`` (note that ``\bar c`` means gradient, and ``c'`` means conjugate). It's exactly as if the complex number were just a pair of reals `(re, im)`. This works out of the box.
+If `f` returns a real number, things are fairly straightforward. For ``c = x + yi`` and  ``z = f(c)``, we can define the adjoint ``\bar c = \frac{\partial z}{\partial x} + \frac{\partial z}{\partial y}i = \bar x + \bar y i`` (note that ``\bar c`` means gradient, and ``c^*`` means conjugate). It's exactly as if the complex number were just a pair of reals `(re, im)`. This works out of the box.
```

Comment:
We should stick with `c'` consistently here, because that's what Julia uses (and it's used in a bunch of other places on this page). If we're using `c*` elsewhere we can change that.

Reply:
It just felt a little bit weird to me, to use `\overline` for derivatives and `'` for the complex conjugate in mathematical notation. `^*` very commonly refers to just the complex conjugate in physics, so it felt more natural here and less confusing. Of course, I left it in snippets that are actual Julia code.

Reply:
Fair, but then we should mention the `'` notation as well. Might be better to have a note on it in context, rather than trying to get it into this paragraph.

Reply:
Where exactly are you talking about?

Reply:
as in, we can clarify the notation when it's first used, rather than trying to define everything in a parenthetical.

Reply:
Ok, will do